src/common/details/console/gui/controls/StdInOutServerHelpers.cs

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+//
+
+using System.Text;
+using System.Text.Json;
+
+namespace Azure.AI.Details.Common.CLI.ConsoleGui
+{
+    public class StdInOutServerHelpers
+    {
+        public static bool IsStdInOutServer => _stdInOutServer != null;
+
+        public static string? QuickInput(string prompt, string? value = null)
```

Comment:
I think "headless" fits better than quick.

Quick makes me wonder "What's slow?" when the design goal is for the non-interactive scenario. And in a class named StdInOut**Server**Helpers headless doesn't feel out of place.

Reply:
I renamed to InOutPipeServer... Changed `QuckInput` to `GetInputFromUser`. Changed QuickPick to `GetSelectionFromUser`